bus/bus.go

```diff
@@ -178,6 +180,62 @@ func (b *Bus) walletSignHandler(jc jape.Context) {
 	}
 }
 
+func (b *Bus) walletSplitHandler(jc jape.Context) {
+	var wsr WalletSplitRequest
+	if jc.Decode(&wsr) != nil {
+		return
+	}
+	if wsr.Outputs == 0 {
+		jc.Error(errors.New("'outputs' has to be greater than zero"), http.StatusBadRequest)
+		return
+	}
+
+	utxos, err := b.w.UnspentOutputs()
```

Comment:
It's interesting to me that we don't lock anything anywhere? Do we consider it good enough that we immediately spend the outputs and just hope that nothing is spending them internally in some data race?

Reply:
yeah this is my bad -- see the other comment thread above